src/Types/TodosWithUsers.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface TodoWithUser {
+  userId: number;
+  id: number;
+  title: string;
+  completed: boolean;
+  user?: User;
+}
```

Comment:
```suggestion
export interface TodoWithUser  extends Todo {
  user?: User;
}
```
In your case if any field changes in todo you have to do it here as well, extend your types and interfaces